crates/ruma-events/src/sticker.rs

```diff
@@ -6,7 +6,33 @@ use ruma_common::OwnedMxcUri;
 use ruma_macros::EventContent;
 use serde::{Deserialize, Serialize};
 
-use crate::room::ImageInfo;
+#[cfg(not(feature = "compat-encrypted-stickers"))]
```

Comment:
I forgot to tell you in the previous review, the feature should actually be without the `not` everywhere